src/plugin/panel/popover.c

```diff
@@ -11,24 +11,42 @@
 
 #include "popover.h"
 
-struct BudgiePopover {
-    GtkWidget * relative_to;
-};
 
-G_DEFINE_TYPE(BudgiePopover, budgie_popover, GTK_TYPE_POPOVER)
+typedef struct {
+	GtkPopover parent_instance;
+} BudgiePopoverPrivate;
```

Comment:
This also seems wrong. I would look at [this class](https://github.com/BuddiesOfBudgie/budgie-desktop/blob/main/src/panel/applets/trash/trash_button_bar.c) in the trash applet, which is also a derivable type with private data.